betelgeuse.py

```diff
@@ -97,6 +98,17 @@ def parse_junit(path):
     return result
 
 
+def parse_test_results(test_results):
+    """Returns the summary of test results by their status.
+
+
+    :param str path: Path to the jUnit XML file.
+    :return: A list of dicts with information about every test
+        case result.
+    """
+    return Counter([test['status'] for test in test_results])
```

Comment:
I saw that a test was added for this. We can do another iteration in the future and start testing the command, once this happens we can merge two functions.